api/environments/identities/tests/test_views.py

```diff
@@ -370,7 +370,7 @@ def test_identities_endpoint_returns_traits(self, mock_amplitude_wrapper):
         trait = Trait.objects.create(
             identity=self.identity,
             trait_key="trait_key",
-            value_type="STRING",
+            value_type=STRING,
```

Comment:
I tend to think this is because those tests were wrong (and we were lucky that the default type is string) but it's worth noting that this is a slight change `"STRING"` -> `"unicode"`.

Reply:
Yep. The engine mapper relies on `Trait`'s `trait_value` property which doesn't employ the default string fallback.